parts/common/components.json

```diff
@@ -752,7 +752,7 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/windows-gmsa-webhook",
```

Comment:
Please update the name since the new gmsa webhook is published by diale.

oss/v2/kubernetes/windows-gmsa-webhook

Reply:
Done.